slack_sdk/web/client.py

```diff
@@ -1315,6 +1315,46 @@ def chat_scheduledMessages_list(self, **kwargs) -> SlackResponse:
         """Lists all scheduled messages."""
         return self.api_call("chat.scheduledMessages.list", params=kwargs)
 
+    def conversations_acceptSharedInvite(
+        self,
+        *,
+        channel_name: str,
+        channel_id: str = None,
+        invite_id: str = None,
```

Comment:
I know some existing methods (e.g., views_update) are not yet consistent about this but let's use `Optional` type hint for the value that can be `None`.
```suggestion
        channel_id: Optional[str] = None,
        invite_id: Optional[str] = None,
```
For existing others, we are happy to change the type hint this time! We don't call it a breaking change. It's a correction that does not affect runtime behavior.

Reply:
Sure thing. I see that the type hint is another way of declaring a union type with None. I've included that for other existing methods that don't currently use the `Optional` type hint as well (including views_update) in the latest changes.